### PR TITLE
Cleans up ezzif so there's no hardcoded #define for dip pins.

### PR DESCRIPTION
**Pull request by RobertBaruch**

### CODE REVIEW

**Comment by RobertBaruch** [comment]
Also fixes a bug where ezzif_o and ezzif_i set the wrong value for tristate.

File: firmware/ezzif.c [hunk 1]
```diff
@@ -214,11 +213,11 @@ void ezzif_io_d40(int n, int tristate, int val) {
 }
 
 void ezzif_o_d40(int n, int val) {
-    ezzif_io_d40(n, 1, val);
```
Comment (JohnDMcMaster) [inline firmware/ezzif.c]: if this is a terrible bug, lets break this out to own PR

**Comment by JohnDMcMaster** [comment]
Some context missing on here: given the relatively tight timing and MCU space, I was hoping to have ezzif be preprocessed (hence the macros) as I'm afraid other solutions may preclude some use. That said, I think there is a place for a dynamic library as well since it will cover many scenarios and is the "ez" approach. The preprocessor thing needs more work / cleanup anyway, so maybe we can just merge this and I'll revisit that later if it becomes relevant.

**Comment by JohnDMcMaster** [comment]
This has at for a while without progress. I'm unfortunately going to close this as won't fix given this wasn't the intent for the code direction. We can revisit later though if of interest.
